src/clientContext.js

```diff
@@ -229,10 +229,15 @@ const getClientContext = async () => {
     "Merging with model inputs from clientContext web extension experiment API",
   );
 
+  safe_total_uri_count = await browser.privileged.clientContext.getTotalUriCount();
```

Comment:
maybe just `safe_total_uri_count = await browser.privileged.clientContext.getTotalUriCount() || 0`

Reply:
Ah yeah - that's much nicer. thanks!

Reply:
Patched in b311391